src/core/adapters/s3Client.ts

```diff
@@ -283,10 +294,16 @@ export async function createS3Client(params: Params): Promise<S3Client> {
 
             const out: ReturnType<S3Client["list"]> = {
                 "directories": [],
-                "files": []
+                "files": [],
+                "errors": []
             };
 
             stream.once("end", () => dOut.resolve(out));
+            stream.on("error", e => {
+                console.log(e.message);
```

Comment:
Forgotten console log

Reply:
don't you think this error can be useful to log ?